client/js/map.js

```diff
@@ -442,7 +442,7 @@ var initialize = function () {
     // Adjust map bounds after it’s fully loaded, but only before first interaction
     google.maps.event.addListener(map, 'tilesloaded', function() {
       if (!firstInteraction &&
-        (token != null || organization != null || treeid != null || userid != null || donor != null)) {
+        (token != null || organization != null || treeid != null || userid === null || donor != null)) {
```

Comment:
Why did we change != to === here? We don't want to set initial zoom to 10 when a userid is specified, as we do for other subsets?

Reply:
this is not a big PR :) below the conditional there is the line which sets the bounds. i thought that's what we want to avoid when there's a userid?